src/__snapshots__/core.test.ts.snap

```diff
@@ -8,12 +8,15 @@ Array [
 // Run \`npx nextjs-routes\` to regenerate this file.
 /* eslint-disable */
 
+// prettier-ignore
+type UnknownQueryParams = { [key: string]: string | string[] | undefined }
```

Comment:
Could you move this into the declare module nextjs-routes below and then import it at the call sites? That way it's not globally available.